src/GLM.jl

```diff
@@ -1,119 +1,151 @@
 module GLM_
 
-import MLJBase
+# -------------------------------------------------------------------
+# TODO
+# - return feature names in the report
+# - return feature importance curve to report using `features`
+# - handle binomial case properly, needs MLJ API change for weighted
+# samples (y/N ~ Be(p) with weights N)
+# - handle levels properly (see GLM.jl/issues/240); if feed something
+# with levels, the fit will fail.
+# - revisit and test Poisson and Negbin regression once there's a clear
+# example we can test on (requires handling levels which deps upon GLM)
+# - test Logit, Probit etc on Binomial once binomial case is handled
+# -------------------------------------------------------------------
 
-export OLSRegressor, OLS,
-       GLMCountRegressor, GLMCount
+import MLJBase
+import Distributions
+using Parameters
 
 import ..GLM
 
-####
-#### REGRESSION TYPES
-####
+export NormalRegressor, OLSRegressor, OLS,
+        BinaryClassifier
 
```

Comment:
You export four models but you define only two structs. Currently the ordinary user will not see these aliases. Also, the proliferation of aliases makes it confusing to review the code : -)